src/term/mod.rs

```diff
@@ -850,10 +863,10 @@ impl ansi::Handler for Term {
     #[inline]
     fn delete_chars(&mut self, count: Column) {
         // Ensure deleting within terminal bounds
-        let count = ::std::cmp::min(count, self.size_info.cols());
-
-        let start = limit(self.cursor.col, Column(0), self.grid.num_cols() - 1);
-        let end = limit(self.cursor.col + count, Column(0), self.grid.num_cols() - 1);
+        let count = cmp::min(count, self.size_info.cols());
```

Comment:
I don't think this coercion is needed since it's effectively covered by L869 below.